app/cdap/components/GridTable/index.tsx

```diff
@@ -304,17 +319,11 @@ export default function GridTable() {
       <ToolBarList
         setShowBreadCrumb={setShowBreadCrumb}
         showBreadCrumb={showBreadCrumb}
-        columnType={columnType}
+        columnType={columnType} // TODO: column type needs to be send dynamically after integrating with transfomations branch
         submitMenuOption={(option, datatype) => {
-          setSnackbar(() => ({
-            open: true,
-            isSuccess: true,
-            message: 'Function Selected',
-          }));
-          return false;
-          // TODO: will integrate with add transformation panel later
+          !transformationOptions.includes(option) ? onMenuOptionSelection(option, datatype) : null;
```

Comment:
is null meant to be returned here? I don't think we want implicit returns, especially if you leave the brackets above.

Implicit returns should be short one liners that look like `() => returnValue` (no brackets).

Reply:
Done